interactive_engine/executor/ir/proto/algebra.proto

```diff
@@ -191,10 +191,16 @@ message Limit {
 // where the values referred by k1, k2, ... are indexed and hence the
 // predicate can be efficiently verified by leveraging the index.
 message IndexPredicate {
+  message PkValue {
```

Comment:
Looks like `PKValue` is not quite necessary here, Simply define 
```
message Triplet {
    common.Property key = 1;
    one of value {
         common.Value const = 2;
         common.DynamicParam param = 3;
    }
}
```

Reply:
Done